tests/opt-testcases/sample.sh

```diff
@@ -27,9 +27,10 @@ run_test    "Sample: ssl_client1, gnutls server, TLS 1.2" \
             -C "error"
 
 requires_protocol_version tls13
```

Comment:
Looks like something we could auto-detect. (Pre-existing, so no change required.)